packages/modules/redis/src/redis-container.ts

```diff
@@ -44,11 +44,22 @@ export class RedisContainer extends GenericContainer {
       ...(this.persistenceVolume ? ["--save 1 1 ", "--appendonly yes"] : []),
     ];
     if (this.imageName.image.includes("redis-stack")) {
+      const existingRedisArgs =
+        (this.createOpts.Env || []).find((e) => e.startsWith("REDIS_ARGS="))?.split("=", 2)[1] || "";
```

Comment:
Potential bug here `?.split("=", 2)[1]`, if the value is `undefined`, then calling `[1]` will throw `TypeError`, would need to be `?.[1]`

Also please use `??` instead of `||` for specifying default values. 

I would use `this.environment` instead of `this.createOps.Env` to simplify this (don't need to split strings).